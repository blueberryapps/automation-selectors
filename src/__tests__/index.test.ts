import { qaID } from '..';

test('ios returns testID', () => {
  expect(qaID('ios', 'test', 'BUTTON')).toMatchSnapshot();
});

test('android returns accessibilityLabel', () => {
  expect(qaID('android', 'test', 'BUTTON')).toMatchSnapshot();
});

test('others (web) returns data-test', () => {
  expect(qaID('anyOther', 'test', 'BUTTON')).toMatchSnapshot();
});