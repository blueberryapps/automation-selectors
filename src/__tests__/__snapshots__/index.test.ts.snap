// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`android returns accessibilityLabel 1`] = `
Object {
  "accessibilityLabel": "BUTTON",
}
`;

exports[`ios returns testID 1`] = `
Object {
  "testID": "BUTTON",
}
`;

exports[`others (web) returns data-test 1`] = `
Object {
  "data-test": "BUTTON",
}
`;
